### ISSUE TITLE
test problem

### ISSUE THREAD

**Issue by Lackl0** [jswati31/gazeclr#3]
Hi, I am very interested in your paper and research content. I have two questions. First, I find that there is only code related to training in your code, but no code related to testing. I wonder if you could share the code related to testing?

**Comment by Lackl0** [comment]
Secondly, I did not find the location of miniEVE, and it seems that the datasets after you processed only train and val datasets, but not test data sets, and there are no parts[1] == 'eye' files in the dataset

**Comment by jswati31** [comment]
Hello, thank you for your interest in our work, and sorry for the delayed response. 

For your first question on testing, we used a linear layer of dim 32 and then 3 to output gaze direction, and input is features output from the GazeCLR model.

MiniEVE is just a subset of the whole EVE dataset and consists of around 5 subjects from the training set. We chose the first 5 subjects as MiniEVE; you can also randomly sample them. 
As mentioned in the paper, we did not use test set of EVE because of missing label information. We evaluated on the val set.
